js/app/Config.js

```diff
@@ -26,6 +26,7 @@ import app from './App.js';
 import md from 'angular-markdown-it';
 import markdownitLinkTarget from 'markdown-it-link-target';
 import markdownitCheckbox from 'legacy/markdown-it-checkbox.js';
+import markdownitAbbr from  'legacy/markdown-it-abbr.js';
```

Comment:
```suggestion
import markdownitAbbr from  'markdown-it-abbr';
```
`markdown-it-abbr` is avaliable in npm so I don't think it's necessary to have the file in this repo.

Reply:
OK, i did not know that. I am not used to npm.

Reply:
That's totally fine!